src/fileops/httpiochain.hpp

```diff
@@ -138,7 +146,7 @@ class HttpIOChain : NonCopyable{
     virtual dav_ssize_t readToFd(IOChainContext & iocontext, int fd, dav_size_t size);
 
     virtual dav_ssize_t preadVec(IOChainContext & iocontext, const DavIOVecInput * input_vec,
-                              DavIOVecOuput * output_vec,
+                             DavIOVecOuput * output_vec,
```

Comment:
Something went wrong with the indentation here

Reply:
Yes, indeed. Easyto fix.